source/contributing/guidelines.rst

```diff
@@ -11,17 +11,20 @@ General steps
 
 1. Setup your workspace as described in :doc:`../../preparing/index`.
 
+#. Make sure you're familiar with Git and GitHub. If your knowledge needs a refresh, take a look here: :doc:`howtogit`
+
 #. Check for existing issues in the `SpongeAPI <https://github.com/SpongePowered/SpongeAPI/issues>`_, `SpongeForge
-   <https://github.com/SpongePowered/SpongeForge>`_, `SpongeVanilla <https://github.com/SpongePowered/SpongeVanilla>`_ and
-   `SpongeCommon <https://github.com/SpongePowered/SpongeCommon>`_  repositories. There is possibly someone else
-   already working on the same thing. You can also check `issues marked with "help wanted"
-   <https://github.com/SpongePowered/SpongeAPI/labels/help%20wanted>`_ for existing issues we need your help with.
+   <https://github.com/SpongePowered/SpongeForge>`_, `SpongeVanilla <https://github.com/SpongePowered/SpongeVanilla>`_,
+   `SpongeCommon <https://github.com/SpongePowered/SpongeCommon>`_ and `SpongeDocs <https://github.com/SpongePowered/SpongeDocs>`_
+   repositories. There is possibly someone else already working on the same thing. You can also check
+   `issues marked with "help wanted" <https://github.com/SpongePowered/SpongeAPI/labels/help%20wanted>`_ for existing
+   issues we need your help with.
 
 .. note::
     Please don't submit pull request for small changes under 20 lines. Instead, `join #spongedev on IRC (irc.esper.net)
-    <TODO>`_ and we'll change it together with the other smaller changes.
+    <irc://irc.esper.net:6667/spongedev>`_ and we'll change it together with the other smaller changes.
```

Comment:
It might be a good idea to use a webchat link here, or link to another docs page with an explanation of IRC.